README.md

```diff
@@ -51,6 +51,7 @@ full fledged examples can be found under `example/`
   - `font` <?[String]> font name to be used. Defaults to `Times New Roman`.
   - `fontSize` <?[Number]> size of font in HIP(Half of point). Defaults to 22. Supports equivalent measure in [pt].
   - `complexScriptFontSize` <?[Number]> size of complex script font in HIP(Half of point). Defaults to 22. Supports equivalent measure in [pt].
+  - `tableRowCantSplit` <?[Boolean]> flag to allow table row to split across pages. Defaults to `false`.
```

Comment:
I recommend changing this to `table: { row: { cantSplit: <Boolean>} }`. This would leave room for future table/row/cell properties without bloating the root level properties

Reply:
updated